auditlog/context.py

```diff
@@ -64,3 +64,15 @@ def _set_actor(user, sender, instance, signal_duid, **kwargs):
             instance.actor = user
 
         instance.remote_addr = auditlog["remote_addr"]
+
+
+@contextlib.contextmanager
+def disable_auditlog():
+    threadlocal.auditlog_disabled = True
+    try:
+        yield
+    finally:
+        try:
+            del threadlocal.auditlog_disabled
+        except AttributeError:
+            pass
```

Comment:
it would be great to have to test to cover this line

Reply:
The attribute error will never happen unless it's wrapped twice for some reason.
Deleting the attribute is covered here: https://github.com/jazzband/django-auditlog/pull/446/files#diff-eb9581b1c35dba2bdbfcf71d8c389814416cdbe08a5dadadc0ce3ab4520b61b2R1846

Is that okay?

Reply:
Ok, let's have it like this